sentry_guide.md

```diff
@@ -0,0 +1,8 @@
+# Sentry: crash reporting
+Each platform has its own documentation for how it uses Sentry:
+- Android ([focus-android](https://github.com/mozilla-mobile/focus-android/wiki/Crash-Reporting-with-Sentry))
+- iOS ([firefox-ios](https://github.com/mozilla-mobile/firefox-ios/wiki/Crash-Reporting-with-Sentry))
+
+## Getting the DSN/API keys for a project
+To download the DSN, open your project in the Sentry dashboard and click here:
+![Where to click to download the DSN](im/sentry_dsn_click.png)
```

Comment:
I don't quite understand where this icon links to

Reply:
@liuche It's a relative path to `im/sentry_dsn_click.png`. Since this file is in `<project-dir>`, the absolute path is `<project-dir>/im/...`.